packages/next/src/cli/next-lint.ts

```diff
@@ -78,7 +78,7 @@ const nextLint = async (options: NextLintOptions, directory?: string) => {
   const pathsToLint = (
     filesToLint.length ? filesToLint : ESLINT_DEFAULT_DIRS
   ).reduce((res: string[], d: string) => {
-    const currDir = join(baseDir, d)
+    const currDir = existsSync(d) ? d : join(baseDir, d)
```

Comment:
```suggestion
    const currDir = path.isAbsolute(d) ? d : join(baseDir, d)
```
I don't think we need to use `existsSync` here if we're just checking for absolute.

Reply:
Thanks for your advice. I fixed it accroding to your review.